crowdnotifier-sdk/src/main/java/org/crowdnotifier/android/sdk/utils/CryptoUtils.java

```diff
@@ -43,15 +56,20 @@ public static synchronized CryptoUtils getInstance() {
 	public EncryptedVenueVisit getEncryptedVenueVisit(long arrivalTime, long departureTime, VenueInfo venueInfo) {
 
 		G2 masterPublicKey = new G2();
-		masterPublicKey.deserialize(venueInfo.getMasterPublicKey());
+		masterPublicKey.deserialize(venueInfo.getPublicKey());
```

Comment:
I don't mind the rename, but it still is the masterPublicKey even if the ProtoBuf field doesn't quite capture that.

Reply:
Okay. We decided to name it consistently with the protobuf specs.

If you prefer, we can rename it back to masterPublicKey. Whatever you prefer.

Reply:
No problem :). Keep it like it is.